runtime/altair/src/lib.rs

```diff
@@ -1350,6 +1352,38 @@ impl pallet_membership::Config for Runtime {
 	type WeightInfo = pallet_membership::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+	#[derive(Clone, PartialEq, Eq, Debug, TypeInfo, Encode, Decode, MaxEncodedLen)]
+	pub const MaxFeedersPerKey: u32 = 10;
+	pub const FirstValueFee: Fee = Fee::Balance(10 * CFG); // TODO: Specify
+}
+
+impl pallet_oracle_feed::Config for Runtime {
+	type FirstValuePayFee = FeeToTreasury<Fees, FirstValueFee>;
+	type OracleKey = OracleKey;
+	type OracleValue = Quantity;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
```

Comment:
Nit: Maybe we want to rename this type to `Timestamp` and the `pallet_oracle_data_collection::Config::Timestamp` to `Time`?

Reply:
I'll think about this. We have complete naming chaos regarding this, haha.

Probably `Time` to `Timer` says better what it means?

I think calling `Timestamp` to `pallet_timestamp` is not the best name, because it creates a lot of confusion when you need to define a timestamp variable. But it's too late for that and the name is on-chain

Reply:
> Probably `Time` to `Timer` says better what it means?

Yeah maybe. But we can also keep as is!

Reply:
I'll leave it as it by now is until we reach some kind of consensus for time naming. Thanks for pointing this out!